src/components/app/MenuButtons/MetaInfo.js

```diff
@@ -170,16 +170,6 @@ class MetaInfoPanelImpl extends React.PureComponent<Props> {
               {formatTimestamp(meta.interval, 4, 1)}
             </div>
           ) : null}
-          {meta.preprocessedProfileVersion ? (
-            <div className="metaInfoRow">
-              <span className="metaInfoLabel">
-                <Localized id="MenuButtons--metaInfo--profile-version">
```

Comment:
Could you also remove this ftl string from our main ftl file?

Reply:
oh right, thanks for reminding me